### PR TITLE
DOC-4819 Update metrics.adoc

### PR DESCRIPTION
**Pull request by KimberlyFields**

### CODE REVIEW

**Comment by plpesvc-ds** [comment]
Build successful! :white_check_mark:
Deploying draft.

**Review by lukasz-antoniak — APPROVED** [review]

File: modules/ROOT/pages/metrics.adoc [hunk 1]
```diff
@@ -51,18 +51,20 @@ This is configured by the {zdm-automation} through the variable `primary_cluster
...
-You can set the interval via the `Error Rate interval` dashboard variable at the top.
-** Read Failure Rate: one `cluster` label with two settings: `origin` and `target`.
-The label that contains data depends on which cluster is currently considered the primary (same as the latency and throughput metrics explained above).
-** Write Failure Rate: one `failed_on` label with three settings: `origin`, `target` and `both`.
+* Request Failure Rates: the number of request failures per interval.
+You can set the interval in the `Error Rate interval` dashboard variable at the top.
+** Connect Failure Rate: one `cluster` label with two settings, `origin` and `target`, that represent the cluster to which the connection attempt failed.
+** Read Failure Rate: one `cluster` label with two settings, `origin` and `target`.
+The label that contains data depends on which cluster is currently considered the primary, the same as the latency and throughput metrics explained above.
+** Write Failure Rate: one `failed_on` label with three settings, `origin`, `target` and `both`.
```
Comment (brian-r-fisher) [inline modules/ROOT/pages/metrics.adoc:59]:
```suggestion
** Write Failure Rate: one `failed_on` label with three settings, `origin`, `target`, and `both`.
```

**Review by brian-r-fisher — APPROVED** [review]

**Comment by plpesvc-ds** [comment]
Build successful! :white_check_mark:
Deploying draft.

**Comment by plpesvc-ds** [comment]
Build successful! :white_check_mark:
Deploying draft.

**Pull request merged**